app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,43 @@
+// Place all the styles related to the Posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+/* ADD POST MODAL */
+#addPostPopup {
+    margin: 4px 7px;
+    text-align:center;
+	#post_short {
+		width: 85%;
+		min-height: 65px;
+		border-radius: 5px;
+		padding-right: 30px;
+		&:focus {
+			outline: none;
+		}
+	}
+	.tag_container {
+		width: 85%;
+		margin: auto;
+	}
+	#post_long {
+		width: 85%;
+		height: 100px;
+		border-radius: 5px;
+		&:focus {
+			outline:none;
+		}
+	}
+	textarea {
+		-webkit-box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.15);
+		box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.15);
+		border: 1px solid #d0d0d0;
+		padding: 8px 8px;
+	}
+	.counter {
+		text-align: right;
+		width: 90%;
+		top: 0;
+		margin: -29px 0px 29px 0px;
+		color: lightgray;
+	}
+}
```

Comment:
just like everywhere else, make sure you think about modularity in CSS.  for example, styling `textarea` is something you would want to do globally - not just inside a dom element with the `#addPostPopup` id.

CSS needs the same level of attention to factoring as code elsewhere - it's one of the worst offenders in codebases I've seen for poor factoring, but it becomes a maintenance nightmare if you don't factor it right.